api/tests/missing_headings.csv

```diff
@@ -0,0 +1,5 @@
+Asset Code,Serial No.,Notes
```

Comment:
why did this file change? it's named `missing_headings`

Reply:
It is a new file, for testing. It checks for missing headings, especially the most important one which is **Model number**. Remember, we need a **model number** and either **asset code** or serial number in a sample CSV file.